tools/clang/tools/dxc/CMakeLists.txt

```diff
@@ -16,6 +16,9 @@ add_clang_executable(dxc
   dxc.cpp
 #  dxr.rc
   )
+if(MSVC)
+    set_property(TARGET dxc APPEND PROPERTY LINK_FLAGS /DEBUG)
```

Comment:
I don't see us doing this for dxcompiler.dll, but it is able to generate pdbs. Can we use the same mechanism that is used by dxcompiler.dll to control pdb generation?

Reply:
It seems there is a global cmake setting file [HandleLLVMOptions.cmake](https://github.com/Microsoft/DirectXShaderCompiler/blob/master/cmake/modules/HandleLLVMOptions.cmake) (line 338) where we have added the option to emit pdb in release mode for dlls. I could update it to also emit pdbs for executables by appropriately setting CMAKE_EXE_LINKER_FLAGS_RELEASE.